chia/timelord/timelord.py

```diff
@@ -1093,6 +1093,9 @@ async def _do_process_communication(
 
         except ConnectionResetError as e:
             log.debug(f"Connection reset with VDF client {e}")
+        except Exception as e:
+            tb = traceback.format_exc()
+            log.warning(f"{type(e)} {e} {tb}")
```

Comment:
i think that if it needs a traceback (because it is an exception we aren't predicting) then it should be a log error.  can also use the `log.exception()` helper.  if there are expected errors then they could be caught in another `except` clause and logged without a traceback and at a lower level as appropriate.

```python
        except Exception:
            log.exception("some useful message here about what we were doing that resulted in this exception")
```